src/vm/ast/parser/mod.rs

```diff
@@ -545,6 +549,9 @@ pub fn parse_lexed(mut input: Vec<(LexItem, u32, u32)>) -> ParseResult<Vec<PreSy
 
     // check unfinished stack:
     if parse_stack.len() > 0 {
+        if let Some((_list, start_line, start_column, _parse_context)) = parse_stack.pop() {
+            debug!("Unfinished stack: {} items remaining starting at ({}, {})", parse_stack.len() + 1, start_line, start_column);
+        }
```

Comment:
Oh thank you for this set of debug logs, Parser errors are PITA to handle at this point.
Instead of `debug!`, would you mind setting a `span` on the `diagnostic`?
```
let mut error = ParseError::new(ParseErrors::ClosingParenthesisExpected);
error.diagnostic.spans.push(Span { start_line, start_column, line_pos, column_pos });
return Err(error)
```

Reply:
Sure, I can do that :+1:

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1850/commits/97ab9390750e600a1ffaf7773e9c9aac30c69ea1